framework/src/systems/NonlinearSystem.C

```diff
@@ -351,7 +351,7 @@ NonlinearSystem::converged()
 {
   if (_fe_problem.hasException() || _fe_problem.getFailNextNonlinearConvergenceCheck())
     return false;
-  if (!_fe_problem.allowInvalidSolution() && _solution_is_invalid)
+  if (!_fe_problem.allowSolutionConverged() && _solution_is_invalid)
```

Comment:
allowInvalidSolution was the better name

"SolutionConverged" should always be allowed, it's a good thing that it is converged

Reply:
`allowInvalidSolution()` has already been used. Change it to `allowInvalidSolutionWithConvergence()`